### DISCUSSION TITLE
Scheduled Maintenance on Wednesdays for ConanCenterIndex Pull Request Service

### DISCUSSION THREAD

**Discussion by uilianries** [conan-io/conan-center-index#20497] [Announcements]
Dear Conan Community,

We want to inform all our valued users about an important change in our continuous integration (CI) service schedule. Starting from 2023, October 11, all Wednesdays will be reserved for internal service maintenance, including necessary improvements and fixes. This maintenance window will allow us to enhance the stability and performance of our CI service and keep you all aware about this scheduled operation.

Here's what you need to know about this scheduled maintenance:

1) Maintenance Window: On Wednesdays, the CI service can be under maintenance for the entire day. During this time, the service will not be available to users. The website https://status.conan.io/ will be always updated with the current situation, please, use it as the main source of the CI status.

2) Not Every Wednesday: It's important to clarify that not every Wednesday will have maintenance. We will conduct maintenance activities on selected Wednesdays to ensure a safer and more predictable maintenance schedule. This approach is designed to minimize unpredictable service disruptions during the rest of the week, allowing users to plan their work with greater confidence.

Please keep this in mind if you are contributing to Pull Requests in Conan Center: The service will automatically launch pending jobs - no action is required from contributors. We kindly request you, do not close and re-open pull requests or any other mechanism to try and jump the queue.

We understand the importance of a reliable ConanCenterIndex service in your development workflow, and we are committed to providing you with the best possible experience. By allocating specific Wednesdays for maintenance, we aim to improve our service without causing unexpected interruptions.

We appreciate your understanding and cooperation as we work to enhance our CI service. If you have any questions or concerns regarding this maintenance schedule, please feel free to reach out to our team to share your thoughts.

Thank you for being a part of our community, and we look forward to delivering an even better CI experience for you.

**Comment by jcar87** [comment]
Closing as this is no longer the case
